app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/react/ManifestEvents.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { get } from 'lodash';
+import { IManifest } from '@spinnaker/core';
+
+interface IManifestEventsProps {
+  manifest: IManifest;
+}
+
+export class ManifestEvents extends React.Component<IManifestEventsProps> {
+  private pillStyle(e: any): string {
+    if (e.type === 'Warning') {
+      return 'alert';
+    } else if (e.type === 'Normal') {
+      return 'success';
+    } else {
+      return '';
+    }
+  }
+
+  public render() {
+    if (this.props.manifest && !this.props.manifest.events) {
+      return <div>No events found - Kubernetes does not store events for long.</div>;
+    }
+    const { events } = this.props.manifest;
+    return events.map((e, i) => {
+      return (
+        <div key={get(e, ['metadata', 'uid'], String(i))} className="info">
+          <div className="horizontal">
+            {e.count && (
+              <div className={`pill ${this.pillStyle(e)}`}>
+                {e.count} × <b>{e.reason}</b>
+              </div>
+            )}
+          </div>
+          {(e.firstTimestamp || e.lastTimestamp) && (
+            <div>
+              {e.firstTimestamp === e.lastTimestamp && (
+                <div>
+                  <i>{e.firstTimestamp}</i>
```

Comment:
This is a feature request that you didn't ask for...but while you're here, could you make this a relative time instead of a timestamp? (see the time formatters functions: https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/utils/timeFormatters.ts#L37-L43). The output (e.g., `two hours ago`) is much more intuitive to me.

Reply:
done

<img width="713" alt="screen shot 2018-05-25 at 3 10 19 pm" src="https://user-images.githubusercontent.com/34253460/40562156-ce25a29c-602d-11e8-928a-6768285dda6b.png">

Reply:
Awesome! Thanks so much.